packages/config/src/globals.ts

```diff
@@ -131,7 +131,7 @@ export const hiddenPrefix = '_#_'
 
 export const clearAll = 'clearAll'
 export const allDbs = 'allDbs'
-export const socketPath = origin.replace('3000', '4000')
+export const socketPath = 'http://localhost:4000'
```

Comment:
@lilitkarapetyan - this will work for Heroku, and for a dev machine.  But, when Serge is running on a LAN, we can't hard-code `localhost`.